pyartcd/pyartcd/util.py

```diff
@@ -49,6 +49,18 @@ def isolate_el_version_in_branch(branch_name: str) -> Optional[int]:
     return None
 
 
+def isolate_major_minor_in_group(group_name: str):
+    """
+    Given a group name, determines whether is contains
+    a OCP major.minor version. If it does, it returns the version value as (int, int).
+    If it is not found, (None, None) is returned.
+    """
+    match = re.fullmatch(r"openshift-(\d+).(\d+)", group_name)
+    if not match:
+        return None, None
+    return int(match[1]), int(match[2])
+
+
 def is_greenwave_all_pass_on_advisory(advisory_id: int) -> bool:
```

Comment:
since you've copied this to elliott, why not just use it from there?
```suggestion
from elliottlib.errata import is_greenwave_all_pass_on_advisory
```
or, remove it from here entirely and just do that import in `pyartcd/pyartcd/pipelines/prepare_release.py`